README.md

```diff
@@ -23,7 +23,7 @@
 
 # About Layer5
 
-[Layer5](https://layer5.io)'s cloud native application and infrastructure management software enables organizations to expect more from their infrastructure. We embrace _developer_-defined infrastructure. We empower developers to change how they write applications, support _operators_ in rethinking how they run modern infrastructure, and enable _product owners_ to regain full-control over their product portfolio.
+[Layer5](https://layer5.io)'s cloud native application and infrastructure management software enables organizations to expect more from their infrastructure. We embrace _developer_-defined infrastructure. We empower developers to change how they write applications, support _operators_ in re-think how they run modern infrastructure, and enable _product owners_ to regain full-control over their product portfolio.
```

Comment:
Thanks for these corrections, @prakashAditya639. 👍 In this case, "rethinking" is grammatically correct. We'll want to leave this one as-is.